### PR TITLE
Add tracking of paint and render times

### PR DESCRIPTION
**Pull request by angus-c**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fusion-plugin-browser-performance-emitter",
   "decsription": "Emit performance stats from the browser",
-  "version": "1.4.3",
+  "version": "1.5.0",
```
Comment (KevinGrandon) [inline package.json]: This should be a separate PR after, and I guess we're doing pre-releases now instead of releases...

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export function buildPaintTimesObject(paintTimes) {
```
Comment (KevinGrandon) [inline src/utils.js]: I think this is missing the flow declaration at the top?

File: package.json (hunk 1 above)
Comment (angus-c) [reply package.json]: We're already on 1.5.0 (https://github.com/fusionjs/fusion-plugin-browser-performance-emitter/releases). Unclear why the diff shows 1.4.3

Comment (KevinGrandon) [reply package.json]: This is fine then, ignore me. Thanks!
